install_config/install/prerequisites.adoc

```diff
@@ -88,22 +88,25 @@ endif::[]
 - Minimum 40 GB hard disk space for the file system containing *_/var/_*.
 
 |xref:../../architecture/infrastructure_components/kubernetes_infrastructure.adoc#node[Nodes]
-a| - Physical or virtual system, or an instance running on a public or private IaaS.
+a| * Physical or virtual system, or an instance running on a public or private IaaS.
 ifdef::openshift-origin[]
-- Base OS: Fedora 21, CentOS 7.1, or RHEL 7.1 or later with "Minimal"
+* Base OS: Fedora 21, CentOS 7.1, or RHEL 7.1 or later with "Minimal"
 installation option, or RHEL Atomic Host 7.2.4 or later.
 endif::[]
 ifdef::openshift-enterprise[]
-- Base OS: RHEL 7.1 or later with "Minimal" installation option, or RHEL Atomic
+* Base OS: RHEL 7.1 or later with "Minimal" installation option, or RHEL Atomic
 Host 7.2.4 or later.
 endif::[]
-- NetworkManager 1.0 or later
-- 1 vCPU.
-- Minimum 8 GB RAM.
-- Minimum 15 GB hard disk space for the file system containing *_/var/_*.
-- An additional minimum 15 GB unallocated space to be used for Docker's storage
+* NetworkManager 1.0 or later
+* 1 vCPU.
+* Minimum 8 GB RAM.
+* Minimum 15 GB hard disk space for the file system containing *_/var/_*.
+* An additional minimum 15 GB unallocated space to be used for Docker's storage
 back end; see xref:configuring-docker-storage[Configuring Docker Storage]
 below.
+
+|External etcd Nodes
+|Minimum 20 GB hard disk space.
```

Comment:
I would say ... "Minimum 20 GB hard disk space for etcd data"...just be a little more explicit.  Fair?